### DISCUSSION TITLE
How do I get notifications about things like Discussions, releases, PRs, etc

### DISCUSSION THREAD

**Discussion by drewcdisc** [cdisc-org/cdisc-rules-engine#313] [Q&A] [answered]
All notifications are configured through GitHub. There are two broad steps in getting your notifications. 

1) Customize **how** you want to receive your notifications.
2) Customize **which** items you wish to receive notifications for.

All settings are described in [this page](https://docs.github.com/en/account-and-profile/managing-subscriptions-and-notifications-on-github/setting-up-notifications/configuring-notifications) at GitHub.

In answer to a question raised earlier, yes you can get notifications for changes in GitHub Discussions. See the above link.

**Comment by chowsanthony** [comment, marked as answer]
See reply by [drewcdisc](https://github.com/drewcdisc)
